lib/dialects/mssql/blocks.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+var _ = require('underscore');
+
+module.exports = function(dialect) {
+	var parentValueBlock = dialect.blocks.get('value');
+
+	dialect.blocks.set('value', function(params) {
```

Comment:
https://github.com/2do2go/json-sql/pull/26#discussion_r116424732

Logic moved from `builder._pushValue` to `blocks.js`. `number`, `string`, and `null`/`undefined` cases are handled by `parentValueBlock(params)` I belive.